subworkflows/local/circrna_discovery.nf

```diff
@@ -0,0 +1,227 @@
+include { FIND_CIRC                        } from '../../modules/local/find_circ/find_circ/main'
+include { FIND_CIRC_FILTER                 } from '../../modules/local/find_circ/filter/main'
+include { CIRIQUANT_YML                    } from '../../modules/local/ciriquant/yml/main'
+include { CIRIQUANT                        } from '../../modules/local/ciriquant/ciriquant/main'
+include { CIRIQUANT_FILTER                 } from '../../modules/local/ciriquant/filter/main'
+include { CIRCEXPLORER2_REFERENCE          } from '../../modules/local/circexplorer2/reference/main'
+include { CIRCEXPLORER2_PARSE              } from '../../modules/nf-core/circexplorer2/parse/main'
+include { CIRCEXPLORER2_ANNOTATE           } from '../../modules/nf-core/circexplorer2/annotate/main'
+include { CIRCEXPLORER2_FILTER             } from '../../modules/local/circexplorer2/filter/main'
+include { CIRCRNA_FINDER_FILTER            } from '../../modules/local/circrna_finder/filter/main'
+include { SEGEMEHL_ALIGN                   } from '../../modules/nf-core/segemehl/align/main'
+include { SEGEMEHL_FILTER                  } from '../../modules/local/segemehl/filter/main'
+include { STAR_ALIGN as STAR_1ST_PASS      } from '../../modules/nf-core/star/align/main'
+include { STAR_ALIGN as STAR_2ND_PASS      } from '../../modules/nf-core/star/align/main'
+include { SJDB as STAR_SJDB                } from '../../modules/local/star/sjdb/main'
+include { STAR_ALIGN as DCC_1ST_PASS       } from '../../modules/nf-core/star/align/main'
+include { STAR_ALIGN as DCC_2ND_PASS       } from '../../modules/nf-core/star/align/main'
+include { SJDB as DCC_SJDB                 } from '../../modules/local/star/sjdb/main'
+include { STAR_ALIGN as DCC_MATE1_1ST_PASS } from '../../modules/nf-core/star/align/main'
+include { STAR_ALIGN as DCC_MATE1_2ND_PASS } from '../../modules/nf-core/star/align/main'
+include { SJDB as DCC_MATE1_SJDB           } from '../../modules/local/star/sjdb/main'
+include { STAR_ALIGN as DCC_MATE2_1ST_PASS } from '../../modules/nf-core/star/align/main'
+include { STAR_ALIGN as DCC_MATE2_2ND_PASS } from '../../modules/nf-core/star/align/main'
+include { SJDB as DCC_MATE2_SJDB           } from '../../modules/local/star/sjdb/main'
+include { DCC                              } from '../../modules/local/dcc/dcc/main'
+include { DCC_FILTER                       } from '../../modules/local/dcc/filter/main'
+include { CIRCEXPLORER2_REFERENCE as MAPSPLICE_REFERENCE } from '../../modules/local/circexplorer2/reference/main'
+include { MAPSPLICE_ALIGN                  } from '../../modules/local/mapsplice/align/main'
+include { CIRCEXPLORER2_PARSE as MAPSPLICE_PARSE } from '../../modules/nf-core/circexplorer2/parse/main'
+include { CIRCEXPLORER2_ANNOTATE as MAPSPLICE_ANNOTATE } from '../../modules/nf-core/circexplorer2/annotate/main'
+include { CIRCEXPLORER2_FILTER as MAPSPLICE_FILTER } from '../../modules/local/circexplorer2/filter/main'
```

Comment:
Could move these long ones to the end for readability (due to the indents), but not important